hrm-domain/lambdas/search-index-consumer/messagesToPayloads.ts

```diff
@@ -59,16 +59,23 @@ const contactIndexingInputData = async (
 ): Promise<ContactIndexingInputData> => {
   let transcript: string | null = null;
 
-  const transcriptEntry = m.message.contact.conversationMedia?.find(isS3StoredTranscript);
-
-  if (transcriptEntry) {
-    const { location } = transcriptEntry.storeTypeSpecificData;
-    const { bucket, key } = location || {};
-    if (bucket && key) {
-      const transcriptString = await getS3Object({ bucket, key });
-      const parsedTranscript: ExportTranscript = JSON.parse(transcriptString);
-      transcript = parsedTranscript.messages.map(({ body }) => body).join('\n');
+  try {
+    const transcriptEntry =
```

Comment:
Would this find a transcript if the contact had a recording and a transcript stored against them? Just future proofing

Reply:
It will as long as there is an entry that results true for `isS3StoredTranscript`. If we plan to have something like `isS3StoredRecordingTranscript`, that would need to be added here as well (unless it is also considered "a transcript" under `isS3StoredTranscript`).